packages/api-clients/open-api/bffApi.yml

```diff
@@ -11,7 +11,7 @@ info:
   x-api-id: an x-api-id
   x-summary: an x-summary
 servers:
-  - url: "/backend-for-frontend/{{version}}"
+  - url: "/backend-for-frontend/0.0"
```

Comment:
same for version in different envs (even though I guess this is not used in code generation)